src/components/map/components/action/layer-action/modify-action/index.js

```diff
@@ -5,7 +5,7 @@ import { VlModifyAction } from "vl-mapactions/dist/vl-mapactions.js";
 /**
```

Comment:
`vl-mapactions/dist/vl-mapactions.js` import should occur before import of `../../../../../../utils/core`